cfbs/commands.py

```diff
@@ -1023,7 +1023,7 @@ def set_input_command(name, infile):
         return 1
     log.debug("Input data for module '%s': %s" % (name, pretty(data)))
 
-    def _compare_dict(a, b, ignore={}):
+    def _compare_dict(a, b, ignore=set()):
```

Comment:
This was and is wrong. The dict/set used as the default value is only created once and then its modifications are kept between the function calls.